src/rubocop.ts

```diff
@@ -7,36 +7,49 @@ import * as vscode from 'vscode';
 import { getConfig, RubocopConfig } from './configuration';
 import * as os from 'os';
 
-const tmpFileName = path.join(os.tmpdir(), '_rubocop.rb');
-
 export class RubocopAutocorrectProvider implements vscode.DocumentFormattingEditProvider {
     public provideDocumentFormattingEdits(document: vscode.TextDocument): vscode.TextEdit[] {
         const config = getConfig();
-        fs.writeFileSync(tmpFileName, document.getText());
-        const onSuccess = () => {
-            const newText = fs.readFileSync(tmpFileName).toString();
-            fs.unlink(tmpFileName);
-            return [new vscode.TextEdit(this.getFullRange(document), newText)];
-        };
         try {
-            const cmd = config.command.split(/\s+/)
-            cp.execFileSync(
-                cmd.pop(),
-                [...cmd, tmpFileName, ...getCommandArguments(tmpFileName), '--auto-correct'],
-                { cwd: getCurrentPath(document.fileName) },
-            );
+            const args = [
+                ...getCommandArguments(document.fileName),
+                "--auto-correct"
+            ];
+            const options = {
+                cwd: getCurrentPath(document.fileName),
+                input: document.getText()
+            };
+            let stdout;
+            if (config.useBundler) {
```

Comment:
I have this same pattern at https://github.com/misogi/vscode-ruby-rubocop/pull/68/files#diff-19a9cf8d2b61b03f6faa4e3e5bc734e9R182, could pull this up somewhere like:
`function exec(command, args, options, isSync: false)`